pkg/auth/auth_openshift.go

```diff
@@ -169,10 +169,15 @@ func (o *openShiftAuth) DeleteCookie(w http.ResponseWriter, r *http.Request) {
 }
 
 func (o *openShiftAuth) logout(w http.ResponseWriter, r *http.Request) {
+	// FIXME: delete access token
```

Comment:
until then, we count with expiration?

Reply:
oops, I think I was removing code and removed a bit too much

Reply:
actually, the code's never been there, let me fix that